### ISSUE TITLE
Loading failed message with no other information available

### ISSUE THREAD

**Issue by dmansfield** [bazelbuild/bazel#1401]
Command:

```
bazel build tensorflow_serving/...
```

Messages:
```
WARNING: Sandboxed execution is not supported on your system and thus hermeticity of actions cannot be guaranteed. See http://bazel.io/docs/bazel-user-manual.html#sandboxing for more information. You can turn off this warning via --ignore_unsupported_sandboxing.
ERROR: Loading failed; build aborted.
INFO: Elapsed time: 0.254s
```
I have modified the WORKSPACE to add a local repository, and one of the BUILD files to include a dependency from this repository.  This is all detailed on a stack overflow question:

http://stackoverflow.com/questions/37799563/how-do-i-register-custom-op-actually-from-syntaxnet-with-tensorflow-serving

When I start introducing intentional typos, or removing expected files I can see the local_repository definition is ok, the reference to it in the deps (@syntaxnet//syntaxnet) is ok, but the actual target specified is not working.  Whatever I put there (currently "parser_ops_cc") it gives the same error, whether a valid target in the referenced BUILD or not.  I've tried all kinds of options to get more verbosity.

Interestingly "--logging 0" and "--logging 6" make absolutely no difference.  This seems a (different) bug.

**Comment by kchodorow** [comment]
What version of Bazel are you using?

**Comment by dmansfield** [comment]
Sorry, had it copied but never pasted in:

```
release: release 0.2.2b
```

**Comment by kchodorow** [comment]
Can you try with 0.3.0?  I think there were a couple of cases where this could happen that were fixed (although I'm unable to reproduce it myself).

**Comment by dmansfield** [comment]
It failed with:

```
ERROR: /opt/tensorflow/serving/tensorflow_serving/example/BUILD:53:1: error loading package '@syntaxnet//syntaxnet': Encountered error while reading extension file 'google/protobuf/protobuf.bzl': no such package '@tf//google/protobuf': error loading package 'external': The repository named 'tf' could not be resolved and referenced by '//tensorflow_serving/example:mnist_inference'.
ERROR: Analysis of target '//tensorflow_serving/example:mnist_inference' failed; build aborted.
```
Perhaps this is the real reason it was failing? Is there a mismatch between some remaining "@tf" reference where it should be "@org_tensorflow" or something? I'm really a bazel newb

I'm trying a clean rebuild of syntaxnet now with 0.3.0 then I'll head back into serving and see if it builds after that.  Not sure if clean is necessary after bazel version upgrade (killing the daemon certainly was - gotcha!).

**Comment by dmansfield** [comment]
I've moved past this by:

1)  ensuring all submodule "tensorflow" instances are updated to master (by removing all submodules and using symlinks to a single copy instead)

2) patching syntaxnet to no longer expect tensorflow to provide a protobuf submodule, e.g change '@tf//google/protobuf:protobuf' to '@protobuf//:protobuf' in a bunch of places

3) patching syntaxnet to use "@org_tensorflow" instead of "@tf"

So the issue I reported originally, bad error reporting by bazel, is indeed fixed in 0.3.0 it seems.  My actual problem need as a fix in syntaxnet.

**Comment by dmansfield** [comment]
FYI, I'm closing this issue because in 0.3.0 a reasonable error is printed.
